src/styles/typography/font.scss

```diff
@@ -91,6 +91,6 @@ span, p, a {
 }
 
 a {
-  color: $color-blue;
+  color: $color-blue-med;
```

Comment:
Where does `$color-blue-med` come from? I do not know this reference, i'm suspecting you mean `color-blue-500`?